Patch by Stephan Aiche to make the doc build.--- a/doc/doxygen/parameters/TOPPDocumenter.C
+++ b/doc/doxygen/parameters/TOPPDocumenter.C
@@ -42,6 +42,8 @@
 	bool errors_occured = false;
 	for (ToolListType::const_iterator it=tools.begin(); it!=tools.end(); ++it)
 	{
+    std::cerr << "Documenting " << it->first << " .. ";
+    
 		//start process
 		QProcess process;
 		process.setProcessChannelMode(QProcess::MergedChannels);
@@ -51,20 +53,28 @@
  		process.start((it->first + " --help").toQString());
 		process.waitForFinished();
 
-		ofstream f((String("output/")+ prefix + it->first + ".cli").c_str());
+		ofstream f;
+    f.open((String("output/")+ prefix + it->first + ".cli").c_str());
 		std::string lines = QString(process.readAll()).toStdString();
-		if(process.error() != QProcess::UnknownError)
+    if(process.error() != QProcess::UnknownError)
 		{
 			// error while generation cli docu
 			f << "Errors occured while generating the command line documentation for " << it->first << "!" << endl;
 			f << "Please check your PATH variable if it contains the path to the " << it->first << " executable." << endl;
       f << "Output was: \n" << lines << endl;
 			errors_occured = true;
+      std::cerr << "failed" << std::endl;
 		}
+    else if(!f.is_open())
+    {
+      std::cerr << "failed" << std::endl;
+      std::cerr << "Could not open file: " << (String("output/")+ prefix + it->first + ".cli") << std::endl;
+    }
 		else
 		{
 			// write output
 			f << lines;
+      std::cerr << "done" << std::endl;
 		}
 		f.close();
 	}
@@ -74,7 +84,7 @@
 int main (int , char** )
 {
 	//TOPP tools
-	ToolListType topp_tools = ToolHandler::getTOPPToolList(true); // include GenericWrapper (can be called with --help without error, even though it has a type)
+	ToolListType topp_tools = ToolHandler::getTOPPToolList(false); // include GenericWrapper (can be called with --help without error, even though it has a type)
 	topp_tools["TOPPView"] = Internal::ToolDescription();
 	topp_tools["TOPPAS"] = Internal::ToolDescription();
 	//UTILS
